ote/src/cljs/ote/views/admin/sea_routes.cljs

```diff
@@ -51,7 +51,8 @@
       3 ::transit/wednesday
       4 ::transit/thursday
       5 ::transit/friday
-      6 ::transit/saturday)))
+      6 ::transit/saturday
+      ::transit/sunday)))
```

Comment:
Didn't go deep into this logic, so just double-checking is defaulting to sunday ok in all cases? It was already handled as index `0`?

Reply:
Yes, it is ok.

If service calendar and service rule is not given we don't have any dates when route is supposed to be active (ajopäivät, don't know what it is in english). This prevents errors when ajopäivät is not given.